static/app/views/dashboards/editAccessSelector.tsx

```diff
@@ -379,3 +440,20 @@ const FilterButtons = styled(ButtonBar)`
   margin-bottom: ${space(0.5)};
   justify-content: flex-end;
 `;
+
+const CollapsedAvatarTooltip = styled('div')`
+  max-height: 200px;
+  overflow-y: scroll;
```

Comment:
I think we want something like `overflow-y: auto` because if you use `scroll`, the scroll bar is always visible even if there isn't any scrolling to do, e.g.
![Screenshot 2025-01-20 at 11 02 29 AM](https://github.com/user-attachments/assets/da083556-5503-4ec6-9cdb-543f7ea7cbbf)